schema/chainInfoSchema.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$id": "/schemas/ChainInfoSchema.json",
+  "description": "Schema for chain infomation",
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "chainName": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "platformChainId": {
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9]+$"
+    },
+    "subnetId": {
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9]+$"
+    },
+    "vmId": {
+      "type": "string",
+      "pattern": "^[a-zA-Z0-9]+$"
+    },
+    "vmName": {
+      "type": "string"
+    },
+    "rpcUrl": {
+      "type": "string",
+      "format": "uri"
+    },
+    "wsUrl": {
+      "type": "string",
+      "format": "uri"
+    },
+    "multicallAddress": {
+      "type": "string",
+      "pattern": "^0x[a-fA-F0-9]{40}$"
+    },
+    "isTestnet": {
+      "type": "boolean"
+    },
+    "networkToken": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "symbol": {
```

Comment:
Discussed with Leart.  We need some extra validation around `symbol`.  For ERC20/ERC721 contracts, we **should require symbol**.  For _other_ contracts, we **should require that there is no symbol**.

We probably need to do some custom validation for this in `validate.js`.